tests/frontend/test-class-wpseo-opengraph-image.php

```diff
@@ -127,6 +129,16 @@ public function test_add_image_as_string() {
 		$this->assertEquals( $this->sample_array(), $class_instance->get_images() );
 	}
 
+	/**
+	 * Tests whether has images is false by default.
+	 */
+	public function test_add_image_with_no_url_key() {
+		$class_instance = $this->setup_class();
+
+		$class_instance->add_image( array( 'link' => '/test.png' ) );
+		$this->assertEmpty( $class_instance->get_images() );
```

Comment:
Suggestion: you could add some extra tests to test more scenarios in the same function: 
	
	$class_instance->add_image( '' );
	$class_instance->add_image( false );

	// It's not array so it should not be added.
	$class_instance->add_image( (object) array( 'url' => '/index.html' ) );
	$class_instance->add_image( array( 'url' => null ) );
	$class_instance->add_image( array( 'url' => '' ) );